sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/UDAQuerySuite.scala

```diff
@@ -156,20 +165,11 @@ abstract class UDAQuerySuite extends QueryTest with SQLTestUtils with TestHiveSi
       (3, null, null)).toDF("key", "value1", "value2")
     data2.write.saveAsTable("agg2")
 
-    val data3 = Seq[(Seq[Integer], Integer, Integer)](
```

Comment:
`data3` not used in the master?

Reply:
Yes, it seems that this was unused prior to this PR.  I had originally created these unit tests on #25024 by copying older UDAF tests, but data3/agg3 must have been used by tests that I didn't copy over.